.github/workflows/iam-policy-generator.py

```diff
@@ -1,35 +0,0 @@
-import json
```

Comment:
This lives here https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/.github/scripts/iam-policy-generator.py